buildah/buildah.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: buildah
+spec:
+  inputs:
+    params:
+    - name: BUILDER_IMAGE
```

Comment:
So we have now a `buildah` image that is continuously update here : https://quay.io/repository/openshift-pipeline/buildah — coming from upstream images recipes (https://github.com/containers/buildah/tree/master/buildahimage). We should be able to use this one as default (and removing the need to have a dockerfile in there)

Reply:
Fantastic!